pom.xml

```diff
@@ -57,19 +57,19 @@
         <!-- Kafka and it's dependencies MUST reflect what the Kafka version uses -->
         <version.kafka>2.3.0</version.kafka>
         <version.kafka.scala>2.12</version.kafka.scala>
-        <version.curator>2.11.0</version.curator>
+        <version.curator>4.2.0</version.curator>
         <version.zookeeper>3.4.14</version.zookeeper>
         <version.jackson>2.9.9</version.jackson>
         <version.org.slf4j>1.7.26</version.org.slf4j>
         <version.log4j>1.2.17</version.log4j>
         <!-- check new release version at https://github.com/confluentinc/schema-registry/releases -->
-        <version.confluent.platform>5.1.2</version.confluent.platform>
+        <version.confluent.platform>5.3.1</version.confluent.platform>
 
         <!-- Databases -->
         <version.postgresql.driver>42.2.8</version.postgresql.driver>
         <version.mysql.server>5.7</version.mysql.server>
-        <version.mysql.driver>8.0.16</version.mysql.driver>
-        <version.mysql.binlog>0.19.1</version.mysql.binlog>
+        <version.mysql.driver>8.0.17</version.mysql.driver>
+        <version.mysql.binlog>0.20.1</version.mysql.binlog>
```

Comment:
Those dependency updates are good, but seem unrelated to JDK 11 migration; could you file a separate issue and change these in a separate commit attributed to that? Helps to highlight the change in the release notes.

Reply:
Yep I can do that!

- [ ] Remove these updates, file separate jira.

Reply:
Yeah I need to add a step to Travis to separately run tests under both jdks and tweak to still only build jdk8 byte code.